content/header/header.md

```diff
@@ -10,8 +10,9 @@ clinic = "votre clinique"
 services = "vos services"
 inventory = "votre inventaire"
 online = "en ligne"
+support = "recommandé par"
```

Comment:
You used a capital letter in english. Since we do not capitalize texts with css, this will stay uncapitalized in french.